jobs/openshift/cluster/integreatly/osd-cluster-integreatly-upgrade.yaml

```diff
@@ -0,0 +1,69 @@
+---
+- job:
+    name: osd-cluster-integreatly-upgrade
+    display-name: 'OSD Cluster Integreatly Upgrade'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'clusterName'
+          default: ''
+          description: '[REQUIRED] The name of the target cluster to upgrade Integreatly against'
+      - string:
+          name: 'upgradeGitBranch'
+          default: 'master'
+          description: '[REQUIRED] The name of the git branch to be used for the Integreatly installer'
```

Comment:
Description is wrong here, it is not for `installer`, but for upgrade

Reply:
Updated.